### PR TITLE
Add two datasets used in Goyal et al. (2020)

### PR DESCRIPTION
**Pull request by YWAN446**
Add data of 11 patients in Young et al. (2020) and 4 patients in Lescure et al. (2020). Both studies had more information in the original papers but the raw data were not published in the original papers.

### CODE REVIEW

File: data/young2020epidemiologic/young2020epidemiologic.yaml [hunk 1]
```diff
@@ -0,0 +1,434 @@
...
+    value: 3.51702
+  - analyte: naso_swab_SARSCoV2
+    time: 10
+    value: negative
+  - analyte: naso_swab_SARSCoV2
+    time: 11
+    value: negative
+  - analyte: naso_swab_SARSCoV2
+    time: 12
+    value: 3.1293
```
Comment (tillahoffmann) [inline data/young2020epidemiologic/young2020epidemiologic.yaml:43]: Should these be raised to the power of ten, e.g., `10 ** 3.1293 = 1346.7903626291`?

Comment (YWAN446) [reply data/young2020epidemiologic/young2020epidemiologic.yaml]: Just fixed them! Thank you.

**Comment by YWAN446** [comment]
Those two studies did not publish their raw data. The raw data, "Viral_Loads.csv", was obtained in the Goyal et al. 2020 paper (https://www.science.org/doi/10.1126/sciadv.abc7112). I uploaded the same raw data in both folders and subset the data in the markdown files.

**Comment by WenfeiXiao** [comment]
Hi Andrew, I just realized that the dataset of 11 patients from Young et al. (2020) aligns with the Study 6 I reviewed last week in the Challenger dataset. I compared data across the supplementary material from Young's paper, Goyal's Viral_Loads.csv, and the Challenger combined dataset (Study 6).

The Ct values in all datasets are identical. However, the Viral_Loads.csv from Goyal et al. (2020) doesn't include the first subject (S1). Interestingly, the viral loads in Viral_Loads.csv and the Challenger combined dataset differ, but they appear proportional.

Additionally, I noticed that both datasets are missing viral load information for blood, urine, and stool samples. I created issue #48 before realizing that it pertains to the same dataset as the Goyal study. As you mentioned, the number of positive stool, blood, and urine samples was small, so should I close that issue?

**Comment by YWAN446** [comment]
Hi Fiona, thank you for pointing this out. I just compared those three data and I think we should use the supplementary material table in the Young paper as the base. From the Viral_Loads.csv data, I back-calculated the standard curve Goyal used:
```14.1147-0.3231*Ct```,
You should be able to use this formula to calculate the log10 concentration of SARS-CoV-2 RNA and then transform it to non-log scale by (10^x). The unit should be gc/swab. For other sample types, I am not sure whether we can use the same standard curve. @tillahoffmann , any suggestions?

The difference between Challenger and Goyal was caused by Challenger transforming the unit to gc/mL to be able to standardize the unit across different studies in his paper.

I also noticed the reference_event in Goyal is the `first date of positive sample` while in Challenger and Young supplemental were 'symptom onset'. Let's use `symptom onset` for our data.
 
Fiona, you can do a PR for Young's paper and I can remove the folder I have for data/young2020epidemiologic in my PR. We will wait for Till's suggestion for the samples from other sample types. After that, you can close issue #48 .

File: data/lescure2020clinical/lescure2020clinical-extraction.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+Raw data ([Viral_Loads](https://github.com/shedding-hub/shedding-hub/blob/main/data/lescure2020clinical/Viral_Loads.csv)), which is stored on [Shedding Hub](https://github.com/shedding-hub), will be loaded and cleaned to match the most updated [schema](https://github.com/shedding-hub/shedding-hub/blob/main/data/.schema.yaml).
+
+```python
+#load the data;
+Lescure2020 = pd.read_csv("Viral_Loads.csv")
+#Subset the data for Lescure et al. 2020;
+Lescure2020 = Lescure2020[Lescure2020["cov_study"]==4]
+#correct some doa errors;
+Lescure2020.loc[(Lescure2020["ID"]=="E3") & (Lescure2020["dao"]==13),"dao"]=14
```
Comment (tillahoffmann) [inline data/lescure2020clinical/lescure2020clinical-extraction.md:33]: What are these corrections?

File: data/lescure2020clinical/lescure2020clinical-extraction.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+Lescure2020.loc[(Lescure2020["ID"]=="E3") & (Lescure2020["dao"]==15),"dao"]=16
+#some data cleaning to match the schema;
+Lescure2020.loc[Lescure2020["cens"]==0,"VL"]=10**Lescure2020.loc[Lescure2020["cens"]==0,"VL"]
+Lescure2020.loc[Lescure2020["cens"]==1,"VL"]="negative"
+#The positive but not quantifiable data are from Figure 2 in the paper;
+Lescure2020.loc[(Lescure2020["ID"]=="E3") & (Lescure2020["dao"].isin([14,16])),"VL"]="positive"
+Lescure2020.loc[(Lescure2020["ID"]=="E5") & (Lescure2020["dao"]==8),"VL"]="positive"
+
+#add data from the patient 2;
```
Comment (tillahoffmann) [inline data/lescure2020clinical/lescure2020clinical-extraction.md:42]: Why is this patient separate from all the other ones in `Viral_Loads.csv`?

File: data/lescure2020clinical/lescure2020clinical-extraction.md [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+#Patient information from Table 1 and Figure 2 in Lescure et al. (2020)
+patients_info = pd.DataFrame(dict(ID=['E1','E2','E3', 'E4', 'E5'],
+                                  age=[31, 48, 80, 30, 46],
+                                  sex=['male','male','male','female','female'],
+                                  day0=[6,9,8,2,2]))
+
+participant_list = [dict(attributes=dict(age=patients_info.loc[patients_info["ID"]==i,"age"].item(),
+                                         sex=patients_info.loc[patients_info["ID"]==i,"sex"].item(),),
+                         measurements=[dict(analyte="naso_swab_SARSCoV2",
+                                             time=int(Lescure2020.loc[j,"dao"].item()+patients_info.loc[patients_info["ID"]==i,"day0"].item()),
```
Comment (tillahoffmann) [inline data/lescure2020clinical/lescure2020clinical-extraction.md:65]: Could be useful to have a small comment here to explain what the `dao` and `day0` variables mean.

**Review by tillahoffmann — COMMENTED** [review]
This looks great, a few small comments just to make sure I understand correctly.

File: data/lescure2020clinical/lescure2020clinical-extraction.md (hunk 3 above)
Comment (YWAN446) [reply data/lescure2020clinical/lescure2020clinical-extraction.md:42]: The `Viral_Loads.csv` is the data from Goyal et al. (2020), which only included 4 patients from Lescure et al. (2020). Data from Patient 2 were added based on Figure 2 in Lescure et al. (2020). I added this in the comment.

File: data/lescure2020clinical/lescure2020clinical-extraction.md (hunk 2 above)
Comment (YWAN446) [reply data/lescure2020clinical/lescure2020clinical-extraction.md:33]: The last two records had incorrect day (13 adn 15) based on Figure 2 in Lescure et al. (2020). We corrected them with days 14 and 16. I also added this explanation in the comment.

File: data/lescure2020clinical/lescure2020clinical-extraction.md (hunk 4 above)
Comment (YWAN446) [reply data/lescure2020clinical/lescure2020clinical-extraction.md:65]: Added a comment: `#the variable "dao" is "the day after first positive" used in Goyal et al. (2020) and "day0" is "the sampling day after onset". "dao" + "day0" will be "day after symptom onset".`

Comment (tillahoffmann) [reply data/lescure2020clinical/lescure2020clinical-extraction.md:68]: Great, thank you!

**Review by tillahoffmann — APPROVED** [review]
Looks ready to merge! Thank you for the explanations.

**Pull request merged**